vscode/microsoft-kiota/src/kiotaInstall.ts

```diff
@@ -6,16 +6,34 @@ import * as admZip from 'adm-zip';
 import * as vscode from "vscode";
 import isOnline from 'is-online';
 
+
+const kiotaInstallStatusKey = "kiotaInstallStatus";
+const installDelayInMs = 30000; // 30 seconds
```

Comment:
Would it make sense to increase this slightly, given that the current binary releases are about 30MB in size, is it too extreme to imagine that potentially a client would be unable to download the file at a rate of 1MB per second, followed by extracting the contents?

Reply:
I don't have a strong opinion here. This logic will only run once per vscode window lifecycle (or if the restart extension command is called). I wanted it to be short in case somebody runs into a problem, closes vs code and re-opens it, so they don't run into a stale lock.